admin/tools/docker/1_build-latest-image.sh

```diff
@@ -84,28 +86,26 @@ DEPS_DIR=$(echo $DEPS_DIR | sed 's%\(.*[^/]\)/*%\1%')
 # So, copy it from templates to SCRIPT_DIR directory
 SCRIPT_DIR="$DOCKERDIR/scripts"
 
-. "$DOCKERDIR"/dist.sh
-TPL_DEPS_SCRIPT="$DEPS_DIR/qserv-install-deps-"$DIST".sh"
+TPL_DEPS_SCRIPT="$DEPS_DIR/qserv-install-deps-debian8.sh"
 
 printf "Add physical link to dependencies install script: %s\n" "$TPL_DEPS_SCRIPT"
 ln -f "$TPL_DEPS_SCRIPT" "$SCRIPT_DIR/install-deps.sh"
 
 # Build the release image
-TAG="fjammes/qserv:$VERSION"
+TAG="qserv/qserv:$VERSION"
 printf "Building latest release image %s from %s\n" "$TAG" "$DOCKERDIR"
-docker build --tag="$TAG" "$DOCKERDIR"
+docker build $CACHE_OPT --tag="$TAG" "$DOCKERDIR"
```

Comment:
There is some inconsistency in `$TAG` expansion in all scripts, in some pace it is used in quotes (`"$TAG"`) and in other places it is used without quotes. We should try to do it consistently, if tag is not supposed to contain spaces then we should not use quotes anywhere.